docking.js

```diff
@@ -1096,6 +1106,24 @@ var DockedDash = GObject.registerClass({
                 this.remove_style_class_name('extended');
             }
         }
+        this._updateVisibleDesktop();
+    }
+
+    _updateVisibleDesktop() {
+        if (!DockManager.settings.get_boolean('intellihide'))
```

Comment:
This was meant to be

```suggestion
        if (!ockManager.settings.get_boolean('intellihide'))
```

Doesn't it?